jsonpickle/unpickler.py

```diff
@@ -607,23 +607,26 @@ def restore_key(key):
 
         return restore_key
 
-    def _restore_from_dict(self, obj, instance, ignorereserved=True):
```

Comment:
Wait actually upon closer inspection, even though the method is marked private (with the single underscore), it would be best not to entirely remove this parameter unless we're in a major release (even renaming it can cause issues). Can you keep this parameter and add another one for restore_dict_items?

Reply:
Shoot I missed this note. I'll tag v3.2.1 with an update that restores `ignorereserved` shortly.